js/Create_Phone_Number.js

```diff
@@ -0,0 +1,11 @@
+// https://www.codewars.com/kata/create-phone-number/train/javascript
+
+function createPhoneNumber(numbers){
+    var numbersArray = numbers;
```

Comment:
1. What is the purpose of this variable assignment?
2. Consider using ES6 variable declarations

Reply:
I'm not sure it this variable are needed maybe i can use numbers only ?
If not i will change this to const.